lib/modules/links/index.js

```diff
@@ -0,0 +1,51 @@
+import ModuleBase from './../../utils/ModuleBase';
+
+const EVENT_TYPE = {
+  Link: 'dynamic_link_received',
+};
+
+/**
+ * @class Links
+ */
+export default class Links extends ModuleBase {
+  static _NAMESPACE = 'links';
+  static _NATIVE_MODULE = 'RNFirebaseLinks';
+
+  constructor(firebaseApp: Object, options: Object = {}) {
+    super(firebaseApp, options, true);
+  }
+
+  get EVENT_TYPE() {
+    return EVENT_TYPE;
+  }
+
+  /**
+   * Returns the link that triggered application open
+   * @returns {*}
```

Comment:
Should return `@returns {*|Promise.<String>}` see `messaging().getToken()`

Reply:
why not only Promise.<String>?

Reply:
If you can be explicit about the type I think it is better

Reply:
I don't see a reason why not. a promise is returned and in the worst case we reject it or the promise value is null.